test/e2e/docker/Dockerfile

```diff
@@ -18,14 +18,12 @@ RUN go get github.com/go-delve/delve/cmd/dlv
 COPY go.mod go.sum ./
 RUN go mod download
 
-# Install BLS library
-COPY third_party ./third_party
-COPY Makefile *.mk ./
-RUN make install-bls
-
 # Copy Tenderdash source
 COPY . .
 
+# Install BLS library
+RUN make install-bls
```

Comment:
Previous code was here to use docker caching and not rebuild bls with every docker run.